### PR TITLE
Fix to use backslash character when platform is windows.

### PR DESCRIPTION
**Pull request by cpauya**
Hi @aronasorman - this fix is for #2843.

I have re-used the `fle_utils.platforms.is_windows()` function and did some minor PEP-8 fixes.

### CODE REVIEW

**Comment by aronasorman** [comment]
@jtamiace can you pull this in and test if it's working?

**Comment by jtamiace** [comment]
sorry, still cannot see the sidebar

**Comment by jtamiace** [comment]
Error type: 

Uncaught Error: Could not find template: 'topics/sidebar'

File: python-packages/fle_utils/handlebars/loader.py [hunk 1]
```diff
@@ -49,12 +53,18 @@ def get_template_module_paths(module_name):
...
     templates = []
 
+    if is_windows():
+        slash_character = "\\"
+    else:
+        slash_character = "/"
+
     for filepath in get_template_module_paths(module_name):
-        template_name = filepath.split("/")[-1].split(".handlebars")[0]
+        template_name = filepath.split(slash_character)[-1].split(".handlebars")[0]
```
Comment (jamalex) [inline python-packages/fle_utils/handlebars/loader.py]: There should be much nicer `os.path` ways to do this, rather than manually splitting.

Comment (cpauya) [inline python-packages/fle_utils/handlebars/loader.py]: Thanks for the tip! :smile:

**Comment by cpauya** [comment]
Hi @jtamiace can you check if this works?

Btw, may I know which versions of Windows and which browser?

I am using Windows 7 and Firefox v34.0.5 on Virtualbox v3.18.x.

**Pull request merged**